include/ocpp/v201/smart_charging.hpp

```diff
@@ -50,6 +53,13 @@ class SmartChargingHandler {
     ///
     ProfileValidationResultEnum validate_evse_exists(int32_t evse_id) const;
 
+    ///
+    /// \brief validates requirements that apply only to the ChargingStationMaxProfile \p profile
+    /// according to the specification
+    ///
+    ProfileValidationResultEnum validate_charge_point_max_profile(const ChargingProfile& profile,
```

Comment:
This is using the 1.6 name; Shouldn't it be:

```suggestion
    ProfileValidationResultEnum validate_charging_station_max_profile(const ChargingProfile& profile,
```

?